server/backends/disk_cache/disk_cache.go

```diff
@@ -409,12 +409,19 @@ func newPartition(id string, rootDir string, maxSizeBytes int64, useV2Layout boo
 	return p, nil
 }
 
+// fileRecord is the data struct we store in the LRU cache
 type fileRecord struct {
 	key       *fileKey
-	lastUse   int64
 	sizeBytes int64
 }
 
+// fileRecordWrapper is a wrapper for fileRecord that contains additional metadata that does not need to be
+// written in the value field to the LRU cache
+type fileRecordWrapper struct {
```

Comment:
golang has a concept of "struct embedding" that makes composing types more convenient by making their fields/funcs accessible without having to reference the embedded type via name. All you need to do is remove the `fileRecord` name so the struct becomes 
type timestampedFileRecord struct {
  *fileRecord
  lastUseNanos int64
}

Reply:
Cool - thanks! Still had to do a lot of explicit "timestampedRecord.fileRecord" because certain functions expect the embedded object itself, as opposed to one of its field/method

Reply:
The `timestampedRecord` is only necessary in `initializeCache`. 
For the other functions, you can create a fileRecord directly. i.e. you can split `makeTimestampedRecord` into `makeTimestampedRecord` and `makeRecord` and use `makeRecord` where you are currently using `makeTimestampedRecord` w/o the lastUse field.

Reply:
Done